plugins/glustershd/rest.go

```diff
@@ -187,3 +190,280 @@ func glustershDisableHandler(w http.ResponseWriter, r *http.Request) {
 
 	restutils.SendHTTPResponse(ctx, w, http.StatusOK, nil)
 }
+
+func glustershInfoHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
The function name is misleading. Lose the gluster prefix and include reference to selfheal in the name.